### ISSUE TITLE
Publish to conancenter

### ISSUE THREAD

**Issue by OmegaRogue** [liblogicalaccess/liblogicalaccess#209]
Publishing this library to conancenter would greatly help with usability, as that would make it possible to use this library like any other library is used with conan

**Comment by Maxhy** [comment]
Agreed. Some work has been done on the Conan recipe already since v2.4, exactly for such purpose. But we didn't complete it to push it on official repo.
It should be straightforward now, just need a bit of additional time for test and completion.
